src/components/SingleArticle.jsx

```diff
@@ -31,7 +38,27 @@ const SingleArticle = () => {
                     <p>Created by {singleArticle.author}</p>
                     <p>{convertTime(singleArticle.created_at)}</p>
                 </div>
-            <p>{singleArticle.body}</p>
+                <p>{singleArticle.body}</p>
+
+                <div className='commments-container'>
+                    <h3>Comments</h3>
+                    {commentsAreLoading ? <p>Loading</p> : comments.map((comment, index) => {
+                        return(
+                            <div className='comment-container' key={index}>
```

Comment:
This could be extracted into a separate component, this could increase readability at a high level